tools/dmiAlphabetizer/alphabetsort.py

```diff
@@ -0,0 +1,189 @@
+#NINX3's dank .dmi file sorter.
+import PIL #pep install pillow
```

Comment:
The requirements are also covered in the readme.md, should I add another file to rehash it again? (I cant find any examples of a requirements.txt, so I dont know if its readme style or some form of automagic system)
The worst that happens without it is the program crashes and you (hopefully?) read the comment.